package.json

```diff
@@ -18,9 +47,14 @@
     "@salesforce/ts-sinon": "1.1.3",
     "@typescript-eslint/eslint-plugin": "^2.30.0",
     "@typescript-eslint/parser": "^2.30.0",
+    "bluebird": "3.5.0",
```

Comment:
Do we really need bluebird (and some of the others)?  I know it's just dev dependencies but it'd be nice to keep it as slim as possible, especially since smoke tests are being removed.

Reply:
Agree, remove everything not needed.